### PR TITLE
Minor fixes to sanitycheck script

### PR DESCRIPTION
**Pull request by nashif**
* sanitycheck: remove obsolete DISABLE_TRYRUN
* sanitycheck: fix documentation and remove mention of kbuild
* sanitycheck: rename qemu_time to handler time
* sanitycheck: use glob to find board yaml file

### CODE REVIEW

File: scripts/sanitycheck [hunk 1]
```diff
@@ -271,16 +271,16 @@ class Handler:
         """Constructor
 
         @param name Arbitrary name of the created thread
-        @param outdir Working directory, should be where qemu.pid gets created
-            by kbuild
-        @param log_fn Absolute path to write out QEMU's log data
-        @param timeout Kill the QEMU process if it doesn't finish up within
+        @param outdir Working directory, should be where handler pid file (qemu.pid for example)
+            gets created by the build system
+        @param log_fn Absolute path to write out hanlder's log data
```
Comment (pfalcon) [inline scripts/sanitycheck]: Typo: "hanlder" (should be "handler").

Comment (pfalcon) [reply scripts/sanitycheck]: And maybe "emulator" or at least "runner" would be more explicit?

File: scripts/sanitycheck [hunk 2]
```diff
@@ -1786,9 +1783,9 @@ class TestSuite:
                         eleTestsuite.remove(tc)
 
             if not goal.failed and goal.qemu:
-                    qemu_time = "%s" %(goal.metrics["qemu_time"])
+                    handler_time = "%s" %(goal.metrics["handler_time"])
```
Comment (pfalcon) [inline scripts/sanitycheck]: I would take a chance for PEP8 fix here and below: space from both sides of `%` operator, (i.e. `"%s" % (goal`)

File: scripts/sanitycheck [hunk 3]
```diff
@@ -1786,9 +1783,9 @@ class TestSuite:
...
 
-            eleTestcase = ET.SubElement(eleTestsuite, 'testcase', classname="%s:%s" %(i.platform.name, i.test.name), name="%s" %(name), time=qemu_time)
+            eleTestcase = ET.SubElement(eleTestsuite, 'testcase', classname="%s:%s" %(i.platform.name, i.test.name), name="%s" %(name), time=handler_time)
```
Comment (pfalcon) [inline scripts/sanitycheck]: PEP8 for `%` operator

File: scripts/sanitycheck (hunk 2 above)
Comment (nashif) [reply scripts/sanitycheck]: well be dealt with in another PR

**Review by tyrantlizard77 — APPROVED** [review]

**Pull request merged**